### DISCUSSION TITLE
warning when u get claims

### DISCUSSION THREAD

**Discussion by lunaritytowny** [TownyAdvanced/Towny#7206] [Q&A] [answered]
my server has ingame ranks bought through playtime and money made while playing, but when I give them claims for ranking up, it gives a big warning about IRL money paying. Is there a way to turn this msg off? i tried editing lang and adding it to global as blank.

**Comment by LlmDl** [comment, marked as answer]
No it is not otherwise the P2W servers violating the Towny license would just turn it off.
